packages/react-components/react-theme/stories/Theme/typography/ThemeTypography.stories.tsx

```diff
@@ -1,51 +1,66 @@
 import * as React from 'react';
-import { teamsLightTheme, typographyStyles } from '@fluentui/react-components';
-import type { TypographyStyle } from '@fluentui/react-components';
-
-const theme = teamsLightTheme;
-
-export const TypographyStyles = () => {
-  // var(--tokenName) => tokenName
-  function formatTypographyStyleValue(typographyStyleValue: TypographyStyle) {
-    return (
-      <div>
-        {Object.values(typographyStyleValue).map(value => (
-          <div key={value}>{value.replace(/var\(--(.+)\)/, '$1')}</div>
-        ))}
-      </div>
-    );
-  }
-
-  // caption1Strong => Caption 1 Strong
-  function formatTypographyStyleName(typographyStyleName: string) {
-    return typographyStyleName.replace(/([A-Z\d])/g, ' $1').replace(/^(.)/, firstChar => firstChar.toUpperCase());
-  }
+import { makeStyles, Subtitle2Stronger, Text, TypographyStyles } from '@fluentui/react-components';
+import { typographyStyles } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  container: {
+    rowGap: '24px',
+    columnGap: '48px',
+    display: 'grid',
+    gridTemplateColumns: 'auto auto 1fr',
+    alignItems: 'start',
+  },
+
+  ...typographyStyles,
+});
+
+export const Typography = () => {
+  const styles = useStyles();
+  const tokenOrder: (keyof TypographyStyles)[] = [
+    'caption2',
+    'caption2Strong',
+    'caption1',
+    'caption1Strong',
+    'caption1Stronger',
+    'body1',
+    'body1Strong',
+    'body1Stronger',
+    'body2',
+    'subtitle2',
+    'subtitle2Stronger',
+    'subtitle1',
+    'title3',
+    'title2',
+    'title1',
+    'largeTitle',
+    'display',
+  ];
+
+  const tokens: [keyof TypographyStyles, string, string[]][] = tokenOrder.map(token => [
```

Comment:
lets use tuple labels for better readability, I have hard time understading what `string` , `string[]` means

```suggestion
const tokens: [token: keyof TypographyStyles, tokenName: string, values: string[]][] = tokenOrder.map(token => [
```

Reply:
Extracted the styles to a separate typing definition. Good suggestion!